dolphinscheduler-master/src/main/java/org/apache/dolphinscheduler/server/master/registry/MasterRegistryClient.java

```diff
@@ -158,10 +154,7 @@ public void removeMasterNodePath(String path, NodeType nodeType, boolean failove
             return;
         }
 
-        String failoverPath = getFailoverLockPath(nodeType, serverHost);
```

Comment:
Why remove lock for failover?

Reply:
The lock is moved to the failoverMasterWithLock method of FailoverService.
```java
private void failoverMasterWithLock(String masterHost) {
        String failoverPath = getFailoverLockPath(NodeType.MASTER, masterHost);
        try {
            registryClient.getLock(failoverPath);
            this.failoverMaster(masterHost);
        } catch (Exception e) {
            LOGGER.error("{} server failover failed, host:{}", NodeType.MASTER, masterHost, e);
        } finally {
            registryClient.releaseLock(failoverPath);
        }
    }
```